subset.py

```diff
@@ -183,6 +170,7 @@ def subset_downstream(self, table, relationships):
         cursor_query ='SELECT DISTINCT * FROM {}'.format(fully_qualified_table(temp_table))
         cursor.execute(cursor_query)
         fetch_row_count = 100000
+        filters = upstream_filter_match(fully_qualified_table(table), columns_query)
```

Comment:
Likewise, upstream filters make sense on upstream tables because we're optionally grabbing data. But downstream tables are only grabbing data based on what has already been grabbed previously, and it doesn't make sense to filter these down anymore. Otherwise we would be removing necessary data and breaking referential integrity, which seems like a weird behavior that breaks the expectation of using condenser in the first place.

Reply:
This applies to all the code down to line 188.